app/views/exercises/apply-renew-passport/apply-renew-passport-alex.html

```diff
@@ -1,6 +1,139 @@
+              href: "#"
+          },
+          {
+              text: "Passports",
+              href: "#"
+          }
+          ]
+      }) }}
+  {% endblock %}
+
+  <main class="govuk-main-wrapper">
+
+    <!-- Content -->
+    {% block content %}
+    
+        <div class="govuk-grid-row">
+
+          <div class="govuk-grid-column-two-thirds">
+
+            <h1 class="govuk-heading-xl">Apply online for a UK passport</h1>
+
+          </div>
+
+        </div>
+
+        <div class="govuk-grid-row">
+
+          <!-- Two thirds -->
+          <div class="govuk-grid-column-two-thirds">
+
+            <p class="govuk-body">Use this service to apply for, renew, replace or update your passport and pay for it online.</p>
+
+            <p class="govuk-body">This service is also available <a href="#">in Welsh (Cymraeg)</a>.</p>
+
+            {{ govukInsetText({
+              text: "If your passport is burgundy or has ‘European Union’ on the cover, you can continue to use it until it expires."
+            }) }}
+
+            <h2 class="govuk-heading-m">How long it takes</h2>
+
+            <p class="govuk-body"><a href="#">Check how long it will take to get a passport</a> before you apply.</p>
+
+            <p class="govuk-body">If you need a passport urgently, you can <a href="#">find out more about the Online Premium or 1 week Fast Track services</a>.</p>
+
+            <p class="govuk-body">Do not book travel until you have a valid passport - your new passport will not have the same number as your old one.</p>
+
+            <h2 class="govuk-heading-m">Before you start</h2>
+
+            <p class="govuk-body">You’ll need a debit or credit card to use this service.</p>
+
+            <p class="govuk-body"><a href="#">It’s £10.50 cheaper</a> to apply for a passport online than by post.</p>
+
+            {{ govukButton({
+              text: "Start now",
+              href: "#",
+              isStartButton: true
+            }) }}
+
+            <h2 class="govuk-heading-m">Other ways to apply</h2>
+
+            <p class="govuk-body">You can pick up passport application forms from your <a href="#">local Post Office</a> and apply by post, or use the <a href="#">Post Office Check and Send service</a>.</p>
+
+            <p class="govuk-body">It takes longer to apply by post than online.</p>
+
+            <div class="margin-top--l border--top border--blue">
+
+              <h3 class="govuk-heading-s">Explore the topic</h3>
+
+              <a href="#" class="govuk-link govuk-!-font-weight-bold">Passports</a>
+
+            </div>
+
+          </div>
+
+          <!-- One third -->
+          <div class="govuk-grid-column-one-third">
+
+            <div class="border--top border--blue">
+
+              <h3 class="govuk-heading-s">Related content</h3>
+              
+              <ul class="govuk-list govuk-list--spaced">
+                <li>
+                  <a class="govuk-link" href="#">Passport fees</a>
```

Comment:
I'd say it's so good that it's almost perfect. But since I can see that you're doing it in such a high standard, I believe you'd make it really perfect and that's why I believe if you apply the [govuk-body-s](https://design-system.service.gov.uk/styles/typography/#body-small) class to the sidebar items then it will be an icing on the cake :) You can check the original page and see if you agree. 

Anyway well done @Alexed93 . You can simply merge it  by yourself or you can update the PR and merge after. It's totally up to you.

Reply:
Dohh, so close yet so far 😉 Thanks Ming! I'll leave it as it is for some fun for any other new starters to try spot.